Controller/ConnectController.php

```diff
@@ -91,14 +94,14 @@ public function registrationAction(Request $request, $key)
             throw new NotFoundHttpException();
         }
 
-        $hasUser = $this->container->get('security.context')->isGranted('IS_AUTHENTICATED_REMEMBERED');
+        $hasUser = $this->container->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_REMEMBERED');
         if ($hasUser) {
             throw new AccessDeniedException('Cannot connect already registered account.');
         }
 
         $session = $request->getSession();
-        $error = $session->get('_hwi_oauth.registration_error.'.$key);
-        $session->remove('_hwi_oauth.registration_error.'.$key);
+        $error   = $session->get('_hwi_oauth.registration_error.' . $key);
```

Comment:
Well, in case there is no standard, the rule would be to leave the code as it is (this makes the review easier by avoiding unnecessary diffs). Introducing a standard should be done in a separate PR doing only that (and after asking the maintainer about the preferred rule)